src/asimo_control/script/camera.py

```diff
@@ -0,0 +1,50 @@
+#This code is modified from : #https://gist.github.com/rethink-imcmahon/77a1a4d5506258f3dc1f
+
+# rospy for the subscriber
+import rospy
+# ROS Image message
+from sensor_msgs.msg import Image
+# ROS Image message -> OpenCV2 image converter
+from cv_bridge import CvBridge, CvBridgeError
+# OpenCV2 for saving an image
+import cv2
+import time
+
+# Instantiate CvBridge
+bridge = CvBridge()
+
+t0 =int(time.time())
+
+def image_callback(msg):
+    print("Received an image!")
+    try:
+        # Convert your ROS Image message to OpenCV2
+        cv2_img = bridge.imgmsg_to_cv2(msg, "bgr8")
+    except CvBridgeError, e:
+        print(e)
+    else:
+        # Save your OpenCV2 image as a jpeg 
+        # if t < 0:
+        t = str(int(time.time())-t0+1)
+        #print(t)
+        # else:
+        if int(t) <= 10:
+            cv2.imwrite('/home/yong_ros/example_ws/src/asimo_e1_description/Photos/camera_image'+t+'.jpeg', cv2_img)
```

Comment:
hardcode that will not work on my laptop

Reply:
Full path is removed.